CMakeLists.txt

```diff
@@ -7,6 +7,8 @@
 
 cmake_minimum_required(VERSION 3.10)
 
+set(CMAKE_TOOLCHAINFILES_PATH ${CMAKE_MODULE_PATH})
```

Comment:
I'd rather see something like what air does, with AIR_RUNTIME_TARGETS.   Basically, we should avoid having a hardcoded toolchain file and support multiple runtime targets from the same build.

Reply:
Furthermore the variable name is misleading, since this isn't a CMAKE variable.